### ISSUE TITLE
mat-select failing accessibility (aria-required-children)

### ISSUE THREAD

**Issue by swatisahay** [angular/components#20909]
Title: WCAG 1.3.1: Ensures elements with an ARIA role that require child roles contain them (#mat-select-0)
Tags: Accessibility, WCAG 1.3.1, aria-required-children

Issue: Ensures elements with an ARIA role that require child roles contain them (aria-required-children - https://dequeuniversity.com/rules/axe/4.0/aria-required-children?application=msftAI)

Element path: #mat-select-0

Expected Behavior
mat-select has role=listbox, so its children should have role=option

Actual Behavior
mat-select children do not have role=option

Environment: Chrome version 86.0.4240.111

====

**Comment by crisbeto** [comment]
The `role="option"` elements are added to the DOM when the select is opened and removed once it's closed. Closing the issue, because in practice it doesn't have an effect over whether the screen reader picks up the options.

**Comment by dman777** [comment]
This still needs attention because lighthouse fails on audits.
